java/test/jmri/jmrit/logix/PackageTest.java

```diff
@@ -32,6 +32,7 @@ public static Test suite() {
         suite.addTest(OBlockTest.suite());
         suite.addTest(OBlockManagerTest.suite());
```

Comment:
This file is the correct location for the OBlock Test suite, and it was already in place.